src/pages/bots/[id]/index.tsx

```diff
@@ -136,29 +143,35 @@ const BotDetail: NextPage = () => {
         <div className="border rounded-lg min-h-[calc(100dvh-240px)] bg-white relative">
           <div className="absolute top-0 flex items-center inset-x-0 border-b h-20 px-4">
             <div className="flex space-x-4">
-              <Avatar src="" />
+              <Avatar src={botLogoSources?.cloudPath ?? ''} />
               <div>
-                <Typography component="b">{botQuery.data?.name}</Typography>
-                <Typography level="p5">Bot description</Typography>
+                <Typography component="b">{sources?.name}</Typography>
+                <Typography level="p5">{sources?.description}</Typography>
```

Comment:
It appears that we are unable to bind the data

![Screenshot 2024-06-03 at 08 51 28](https://github.com/dwarvesf/chatbot-builder/assets/12707960/6d18144d-3c91-442d-9b09-74e83ed73e24)

Reply:
We can check if the description is an empty string, and we can return "Our bot answers instantly."

Reply:
Sorry, it's my fault. It would be sources?.widgetName and sources?.widgetSubheading from the field of bot appearances.